script/aria.js

```diff
@@ -368,18 +367,8 @@ function ariaAttributeReferences() {
                     req +
                     '</li>';
                 }
-                if (node.textContent.indexOf('prohibited') !== -1) {
```

Comment:
GitHub diff viewer looks jumbled below due to line-by-line diffs, but this was a removal of the defunct ``nameFrom: encapsulation` and `nameFrom: legend` blocks, along with the blocks for `nameFrom: heading`… Viewer is interspersing those changes.